x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_logic.ts

```diff
@@ -230,14 +231,14 @@ export const CredentialsLogic = kea<CredentialsLogicType>({
     showCredentialsForm: () => {
       clearFlashMessages();
     },
-    initializeCredentialsData: () => {
-      actions.fetchCredentials();
-      actions.fetchDetails();
-    },
-    fetchCredentials: async (page = 1) => {
+    fetchCredentials: async () => {
       try {
         const { http } = HttpLogic.values;
-        const query = { 'page[current]': page };
+        const { meta } = values;
+        const query = {
+          'page[current]': meta.page.current,
+          'page[size]': meta.page.size,
```

Comment:
See above comment - passing `page[size]` overrides the default API size & allows us to potentially one day allow users to customize table page size if needed